Makefile

```diff
@@ -670,7 +669,7 @@ test-cover: test ## Run tests with code coverage and generate reports.
 
 .PHONY: kind-create-bootstrap
 kind-create-bootstrap: $(KUBECTL) ## Create capz kind bootstrap cluster.
-	export KIND_CLUSTER_NAME=capz-e2e && ./scripts/kind-with-registry.sh
+	export AZWI=$${AZWI:-true} KIND_CLUSTER_NAME=capz-e2e && ./scripts/kind-with-registry.sh
```

Comment:
do bash defaults work as expected in make targets? Asking because I don't see any other occurrences in the Makefile

Reply:
I tested every permutation of this locally (`AZWI unset/true/false` vs. `make kind-create, kind-create-bootstrap`) and saw what I expected to see each time, so I'm inclined to say yes.

Reply:
> what is the double $$ doing?

`make` also uses `$` to denote variables, so if you want to pass a variable expression through to the shell for expansion, you have to double-dollar it.